docs/sources/operations/observability.md

```diff
@@ -5,8 +5,8 @@ weight: 20
 # Observing Grafana Loki
 
 Both Grafana Loki and Promtail expose a `/metrics` endpoint that expose Prometheus
-metrics. You will need a local Prometheus and add Loki and Promtail as targets.
-See [configuring
+metrics (the default port is 3100). You will need a local Prometheus and add
```

Comment:
I am wondering if the default port for Promtail is 80, not 3100. 

Wanted to make a note to not to forget to check this when I get a chance or if someone has a chance sooner.

Reply:
Ah, I think you are right, it seems the listen port for promtail comes from the weaveworks package:

```
$ clients/cmd/promtail/promtail
...
  -server.http-listen-port int
        HTTP server listen port. (default 80)
...
```

I'll update this.